fdbserver/tester.actor.cpp

```diff
@@ -932,6 +932,7 @@ ACTOR Future<Void> clearData(Database cx, Optional<TenantName> defaultTenant) {
 				break;
 			}
 			tr.reset();
+			tr.debugTransaction(debugRandom()->randomUniqueID());
```

Comment:
It might be a little clearer and robust to call this function at the top of every retry loop. Then we could have one call per transaction rather than 2-3 and it would be similar to how we set normal transaction options

Reply:
Make sense. Will update the three transactions in this function as suggested

Reply:
Done